metadata.json

```diff
@@ -12,7 +12,12 @@
   "dashboardTemplateNames": [
     "Template"
   ],
+  "apiUrl": "v1",
   "apiSpecificationPath": "/v1/version/openapi.json",
```

Comment:
/v1 needs to be removed. This information is now in apiUrl. The right value is /version/openapi.json